compiler/src/dmd/declaration.h

```diff
@@ -703,7 +703,7 @@ class FuncDeclaration : public Declaration
     Expressions *fdensureParams(Expressions *fdep);
     bool equals(const RootObject * const o) const override final;
 
-    int findVtblIndex(Dsymbols *vtbl, int dim);
+    int findVtblIndex(FuncDeclaration *fd, DArray<Dsymbol*> vtbl);
```

Comment:
This looks wrong to me. It's a free function now.

Reply:
deleted